### PR TITLE
Request enable

### LINKED ISSUE #34: Enable storage and retrieval of historical data
Create api enpoints for 
- [x] trip
- [x] request

Link trip and request tables together

### PR DESCRIPTION
**Pull request by RetynaGirl**
Updates API contract from FE to BE. MUST BE MERGED ALONG SIDE OF FRONTEND PR #90

Closes #34 

- [x] Sad Path Tested

- [x] Happy Path Tested

- [x] All Tests are passing

- [ ] Updated README.md with added information

- [x] Collabotors:
- Brian Liu

### CODE REVIEW

**Review by cpfergus1 — APPROVED** [review]

**Review by cpfergus1 — APPROVED** [review]

File: app/controllers/api/v1/search_controller.rb [hunk 1]
```diff
@@ -1,18 +1,55 @@
...
+      return
+    end
+
+    trips.each { |trip| request.trips << trip }
+
+    if request.trips.empty?
       payload = {
-          error: trips[:message].first[:errors].first,
-          status: 400
+        error: 'There were no available trips that matched your search.',
```
Comment (stillsheryl) [inline app/controllers/api/v1/search_controller.rb:21]: Nice job making a more detailed response for the user!

File: app/facades/weather_facade.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class WeatherFacade
+  def self.get_updated_weather(lat, long)
+
```
Comment (stillsheryl) [inline app/facades/weather_facade.rb:3]: What's happening here?

Comment (stillsheryl) [reply app/facades/weather_facade.rb:3]: Ooohhh, ok, I see it in PR #37

**Review by stillsheryl — APPROVED** [review]
This looks awesome! Can you please also add to the README sometime today?

**Comment by RetynaGirl** [comment]
@stillsheryl yeah lol, I just needed to get it done and go to bed

**Review by dunlapww — APPROVED** [review]

**Pull request merged**